spec/components/app_vaccination_record_summary_component_spec.rb

```diff
@@ -327,4 +327,242 @@
       expect(rendered).to have_css(".app-highlight", text: "1 January 2027")
     end
   end
+
+  describe "#incorrect_vaccine_method?" do
```

Comment:
We tend to avoid using doubles / mocks for testing view components (see other specs in this file), as they're quite brittle – best use object factories instead.